go.mod

```diff
@@ -4,11 +4,11 @@ go 1.21
 
 require (
 	github.com/pkg/errors v0.9.1
-	github.com/rook/kubectl-rook-ceph v0.6.1-0.20231211183203-70ede2704719
-	github.com/rook/rook v1.13.0
+	github.com/rook/kubectl-rook-ceph v0.7.1-0.20240129162458-c33feeeaf0fb
```

Comment:
This includes all the latest commits to the upstream repo, right?

Reply:
yes